config/libc/bionic.in

```diff
@@ -0,0 +1,185 @@
+# bionic options
+
+## depends on ! WINDOWS && ! BARE_METAL
+## depends on ARCH_arm || ARCH_mips || ARCH_x86
```

Comment:
Since the automatically generated config options are sorted alphabetically, bionic now becomes the default on these architectures. So, where a sample was using glibc by default, it now builds using bionic. This is an ugly kind of dependency in crosstool-ng, but I haven't got to weeding it out yet.

Please update the affected samples.